spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,36 +32,36 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.0-RC1]]
-=== MicroProfile 4.0 Release Candidate 1 (Q42020)
-// MicroProfile 4.0-RC1 is the 14th platform release for the
+[[microprofile4.0]]
+=== MicroProfile 4.0 (Q42020)
```

Comment:
After comments from Today's meeting, maybe (Q12021) is going to be the correct quarter?

Reply:
Agreed, done